_episodes/20-crop-raster-data.md

```diff
@@ -0,0 +1,218 @@
+---
+title: "Crop raster data with rioxarray and geopandas"
+teaching: 40
+exercises: 20
+questions:
+- "How can I crop my raster data to the area of interest?"
+objectives:
+- "Crop raster data with a bounding box."
+- "Crop raster data with a polygon."
+- "Crop raster data within a geometry buffer."
+keypoints:
+- "Use `clip_box` in `DataArray.rio` to crop raster with a bounding box."
+- "Use `clip` in `DataArray.rio` to crop raster with a given polygon."
+- "Use `grow` in `GeoDataFrame` to make a buffer polygon of a (multi)point or a polyline. This polygon can be used to crop data."
+---
+
+It is quite common that the raster data you have in hand is too large to process, or not all the pixels are relevant to your area of interest (AoI). In both situations, you should consider cropping your raster data before performing data analysis. 
+
+In this episode, we will introduce how to crop raster data into the desired area. We will use one Sentinel-2 image over Amsterdam as the example raster data, and introduce how to crop your data to different types of AoIs.
+
+> ## Introduce the Data
+>
+> Raster data: A Sentinel-2 raster image of Amsterdams: `S2_amsterdam.tif`.
+> 
+> Vector data: we will use three different types of vector data as AoIs:
+> - Crop field polygons in north Amsterdam. ([source](https://www.pdok.nl/introductie/-/article/basisregistratie-gewaspercelen-brp-)): `data/crop_fields`.
```

Comment:
Should the `source` link to the download page instead of the introduction? https://www.pdok.nl/downloads/-/article/basisregistratie-gewaspercelen-brp-